### ISSUE TITLE
Consider adding VM implementation

### ISSUE THREAD

**Issue by marchof** [foojay2020/discoapi#15]
It might be an interesting information what VM implementation is shipped with the package. There is

* Hotspot
* GraalVM
* JRockit
* J9

I like the metadata which is provided in Raphael Winterhalter's list:

https://rafael.codes/openjdk/

**Comment by HanSolo** [comment]
Yep that might be something to consider, the only problem is we have to add that data manually per distribution because not every distro offers that kind of information. So I will discuss internally. Thx for the hint.

**Comment by joschi** [comment]
While not all vendors explicitly mention the VM implementation it's rather safe to assume Hotspot by default and then handle any deviations accordingly.

```
# curl -s 'https://joschi.github.io/java-metadata/metadata/vendor/adoptopenjdk/all.json' -H 'accept: application/json' | jq -r .[].jvm_impl | sort -u
hotspot
openj9
```

```
# curl -s 'https://joschi.github.io/java-metadata/metadata/all.json' -H 'accept: application/json' | jq -r .[].jvm_impl | sort -u
graalvm
hotspot
openj9
```
